test/e2e/specs/delete-prohibited/recordset/facet-presentation.spec.ts

```diff
@@ -0,0 +1,249 @@
+
+    await page.goto(`${baseURL}${PAGE_URL}`);
+    await RecordsetLocators.waitForRecordsetPageReady(page);
+  });
+
+  await test.step('should have 1 row visible', async () => {
+    await expect.soft(RecordsetLocators.getRows(page)).toHaveCount(testParams.defaults.numRows);
+  });
+
+  await test.step('verify the text is truncated properly on load, then not truncated after clicking "more"', async () => {
+    // default config: maxRecordsetRowHeight = 100
+    // 100 for max height, 10 for padding, 1 for border
+    const cellHeight = 110;
+    let testCellDimensions;
+
+    const firstRow = RecordsetLocators.getRows(page).nth(0);
+    const testCell = RecordsetLocators.getRowCells(firstRow).nth(2);
+
+    await expect.soft(testCell).toContainText('... more');
+    testCellDimensions = await testCell.boundingBox();
+    expect.soft(Math.trunc(testCellDimensions?.height || 0)).toBe(cellHeight);
+
+    await testCell.locator('.readmore').click();
+
+    await expect.soft(testCell).toContainText('... less');
+    testCellDimensions = await testCell.boundingBox();
+    expect.soft(testCellDimensions?.height).toBeGreaterThan(cellHeight);
+  });
+
+  await test.step('should have ' + testParams.totalNumFacets + ' facets', async () => {
+    await expect.soft(RecordsetLocators.getAllFacets(page)).toHaveCount(testParams.totalNumFacets);
+
+    const titles = RecordsetLocators.getFacetTitles(page);
+    const count = await titles.count();
+    for (let i = 0; i < count; i++) {
+      await expect.soft(titles.nth(i)).toHaveText(testParams.facetNames[i])
+    }
+  });
+
+  await test.step('should have 3 facets open', async () => {
+    await expect.soft(RecordsetLocators.getOpenFacets(page)).toHaveCount(testParams.defaults.openFacetNames.length);
+
+    const titles = RecordsetLocators.getOpenFacetTitles(page);
+    const count = await titles.count();
+    for (let i = 0; i < count; i++) {
+      await expect.soft(titles.nth(i)).toHaveText(testParams.defaults.openFacetNames[i])
+    }
+  });
+
+  // test defaults and values shown
+  await test.step('"id" facet should have 1 facet option checked', async () => {
+    // use 0 index
+    const facet = RecordsetLocators.getFacetById(page, 0);
+    await expect.soft(RecordsetLocators.getFacetSpinner(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getCheckedFacetOptions(facet)).toHaveCount(1);
+  });
+
+  await test.step('"int_col" facet should not show the histogram with 1 facet options checked', async () => {
+    // use 1 index
+    const facet = RecordsetLocators.getFacetById(page, 1);
+    await expect.soft(RecordsetLocators.getFacetSpinner(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getFacetHistogram(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getCheckedFacetOptions(facet)).toHaveCount(1);
+  });
+
+  await test.step('should have 2 filters selected', async () => {
```

Comment:
Let's add a condition to only run this step locally. I'm not sure why this is failing on CI either.